src/Features/Core/Portable/EmbeddedLanguages/RegularExpressions/LanguageServices/RegexPatternDetector.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This seems to *incorrectly* remove the BOM character at the beginning of the file.

Reply:
I don't know why I cannot accept your suggestion (buttons are disabled). I've fixed it manually.